EXTENSIONS.md

```diff
@@ -0,0 +1,75 @@
+# IBM Generative AI Extensions Guide
+Open-source contributors are welcome to extend IBM Gen AI functionality via extensions that must keep the IBM Gen AI package as a dependency. This document explains what extensions are, the types of extensions that exist, and how to create them.
+
+# Table of Contents
+- [About the core package](#ibm-generative-ai-core-package)
+- [What is an extension?](#what-is-an-ibm-generative-ai-extension)
+- [Types of extensions](#ibm-generative-ai-extension-types)
+- [Extensions migration](#extensions-migration)
+- [Design considerations](#design-considerations-to-build-extensions)
+
+# IBM Generative AI Core Package
+[IBM Generative AI open-source repository](https://github.com/IBM/ibm-generative-ai) lives under [IBM Github organization](https://github.com/IBM/). This repository is the official and unique location of IBM Gen AI core package source code.
+
+# What is an IBM Generative AI Extension?
+An extension is a software add-on that is installed on a program to enhance its capabilities. Gen AI extensions are meant to expand the capabilities of the Gen AI core package. For instance, imagine that you want to build a prompt pattern with data from a pandas dataframe. You could write your own code  to achive such functionaliy, or you could instead use the Gen AI Pandas extension. Using the extension will allow you to save development time and deliver an elegant solution that re-uses code when it is possible.
+
+# IBM Generative AI Extension types
+IBM Generative AI supports two types of open-source extensions: open-source official, and open-source third-party extensions.
+
+## Open-source "Gen AI official" extensions
+Extensions that are meant for public use from get-go should instead be developed as open-source extensions. Examples of open-source extensions that have been already released as open-source are LangChain, Pandas, and Hugging Face extensions.
```

Comment:
Is this a better way of separating supported open-source extension and 3rd-party ones?
```suggestion
Extensions that are meant for public use from the get-go should instead be developed as official open-source extensions. Examples of official extensions that have already been released are LangChain, Pandas, and Hugging Face extensions.
```

Reply:
Thanks, I agree.